indico/core/db/sqlalchemy/locations.py

```diff
@@ -202,16 +202,26 @@ def venue_name(self):
     def venue_name(self, venue_name):
         self.own_venue_name = venue_name
 
-    @property
-    def room_name(self):
-        """The name of the room where this item is located."""
+    def get_room_name(self, full=True):
```

Comment:
@nop33 didn't i comment something on one of your PRs about using `room_name` vs `room.name`? I think once this is merged you could simply use `get_room_name(full=False)`